main.go

```diff
@@ -43,9 +43,9 @@ func main() {
 		}
 	}
 	if swaggerFile != "" {
-		data, err := os.ReadFile(swaggerFile)
-		if err != nil {
-			log.Fatalf("cannot read swagger file %s: %v", swaggerFile, err)
+		data, err2 := os.ReadFile(swaggerFile)
```

Comment:
We can disable the strictness of [this](https://github.com/kubewarden/k8s-objects-generator/pull/77/commits/ceb4b7e43c2d7b0e9fe92c1b007051c606861558#r2054240633) lint by changing this to false.
The rationale is that we don't need to be strict, and it is ok to shadow an error. 
After that, we can drop this commit.